ios/FluentUI.Demo/FluentUI.Demo/Demos/ObjectiveCDemoController.m

```diff
@@ -135,7 +135,8 @@ - (void)setupTitleView {
                           subtitle:nil
                    interactivePart:MSFTwoLineTitleViewInteractivePartTitle
                      accessoryType:MSFTwoLineTitleViewAccessoryTypeNone
-       customSubtitleTrailingImage:nil];
+       customSubtitleTrailingImage:nil
+isTitleImageLeadingForTitleAndSubtitle:false];
```

Comment:
Why do we need a bool? Can't we just show/hide the image based on whether the leading image is nil?

Reply:
We can't do that since titleImage being non nil doesn't tell us if it only gets applied to the title or to both title and subtitle